test/clj/rems/test_form_validation.clj

```diff
@@ -118,11 +118,11 @@
 
   (testing "error: field input option can be left empty when optional"
     (is (= nil
-           (validate-fields [{:field/id       "1"
-                              :field/title    {:en "Option list."
+           (validate-fields-for-submit [{:field/id "1"
+                              :field/title         {:en "Option list."
                                                :fi "Valintalista."}
```

Comment:
indentation looks funny here:
- `:fi` should be below `:en`
- we've typically not aligned map values with each other

Reply:
Fixed